general/server/plugins/index.md

```diff
@@ -13,17 +13,15 @@ Jellyfin has a collection of optional plugins that can be installed to provide a
 
 Many plugins are available in a repository hosted on our servers, which can be easily installed using the plugin catalog in the settings. At the moment many of these are still being updated frequently so the version number may not be accurate. There are several different categories that can indicate what kind of functionality the plugins may provide.
 
-**Note to Windows Users:**
-Due to currently unresolved permission issues on Jellyfin Windows installs it is not possible to update and/or uninstall plugins from the UI.
-To update and/or uninstall plugins you must stop Jellyfin, navigate to the local *plugins folder* and delete the `.dll` files for the plugins you want to update and/or uninstall.
+Once a plugin has been installed, removed, or upgraded, a restart of Jellyfin is required in order for the changes to take effect.
 
 The *plugins folder* is located in different locations depending on your install:
 
 * `%UserProfile%\AppData\Local\jellyfin\plugins` for direct installs
```

Comment:
this feels windows specific, we should specify each system. the linux path is buried in a paragraph

Reply:
Agreed, I'm not sure what the correct paths would be though.

Reply:
The 2 paths there are 100% windows specific (IIRC I touched these Paths of this file some time ago due to some Reddit discussion I was part of).
Regardless, adding the Linux paths should be easy right?